modules/amm-executor/src/main/scala/org/ergoplatform/dex/executor/amm/config/BacklogConfig.scala

```diff
@@ -0,0 +1,17 @@
+package org.ergoplatform.dex.executor.amm.config
+
+import derevo.derive
+import derevo.pureconfig.pureconfigReader
+import tofu.Context
+import tofu.logging.derivation.loggable
+
+import scala.concurrent.duration.FiniteDuration
+
+@derive(pureconfigReader, loggable)
+final case class BacklogConfig(
+  orderLifetime: FiniteDuration,
+  orderExecutionTime: FiniteDuration,
+  suspendedProbability: Int
```

Comment:
Is not it better to name it like: `suspendedOrderProbabilityPercent`

Reply:
rename